pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -416,6 +416,68 @@ func (r *ReconcileHyperConverged) Reconcile(request reconcile.Request) (reconcil
 		// One shortcoming is that only one failure of a particular condition can be
 		// captured at one time (ie. if KubeVirt and CDI are both reporting !Available,
 		// you will only see CDI as it updates last).
+
+		if cond, ok := r.conditions[conditionsv1.ConditionDegraded]; ok {
+			if _, ok = r.conditions[conditionsv1.ConditionProgressing]; !ok {
+				r.setStatusCondition(conditionsv1.Condition{
+					Type:    conditionsv1.ConditionProgressing,
+					Status:  corev1.ConditionTrue,
+					Reason:  cond.Reason,
+					Message: cond.Reason,
```

Comment:
So basically we are replicating here, on the progressing condition, the error message we got from the degraded condition of another operator.
I'm a bit scared that this can be even more confusing at user eyes.